src/integration/pages/frontPage/arabic.test.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @pathname /arabic
+ */
+import runCanonicalUserTests from './user.canonical';
+import runAmpUserTests from './user.amp';
+import {
+  runFooterTests,
+  runHeaderTests,
+  runCommonSeoTests,
+  runCommonCanonicalAnalyticsTests,
+  runCommonAmpAnalyticsTests,
+  runCommonA11yTests,
+  runCoreAmpTests,
+  runCoreCanonicalTests,
+  runPerformaceTests,
+} from '../../common';
+
+describe('When I am using the website', () => {
+  runAmpUserTests();
+});
+
+describe('When the application starts', () => {
+  runCoreAmpTests();
+});
+
+describe('When I am analysing user/performance metrics', () => {
+  runCommonAmpAnalyticsTests();
+});
+
+describe('When I am analysing user/performance metrics', () => {
+  runCommonCanonicalAnalyticsTests();
+});
+
+describe('When the application starts', () => {
+  runCoreCanonicalTests({ service: 'arabic' });
+});
+
+describe('Given I am on a Arabic Frontpage AMP/Canonical page', () => {
+  describe('When I am using the website', () => {
+    runHeaderTests({
+      skipToContentText: 'إذهب الى المحتوى',
+    });
+    runCanonicalUserTests();
+    runFooterTests({
+      copyrightText:
+        '© 2020 بي بي سي. بي بي سي ليست مسؤولة عن محتوى المواقع الخارجية. سياستنا بخصوص الروابط الخارجية.',
+      brandingLink: '/arabic',
+    });
+  });
+
+  // To be enabled when the radio schedule is rendered server side
+  // describe('When I am looking for radio schedule component', () => {
+  //   runComponentRenderTest();
```

Comment:
This is a test aimed at users so should be called `runUserTests`

Reply:
Sorry my fault that doesn't need to be under separate describe block `runCanonicalUserTests()` will cover the radio schedule test.